docs/pages/guides/pricing.mdx

```diff
@@ -0,0 +1,130 @@
+account. Each `userId` will count as 1 MAU. To make sure you begin to track MAU
+as intended, first update `@liveblocks/node`.
+
+```bash
+npm update @liveblocks/node@1.0
+```
+
+Then, update your `authorize` function to include the `userId` field.
+
+```ts highlight="16,20"
+import { authorize } from "@liveblocks/node";
+
+// Replace this key with your secret key provided at
+// https://liveblocks.io/dashboard/projects/{projectId}/apikeys
+const secret = "sk_prod_xxxxxxxxxxxxxxxxxxxxxxxx";
+
+export default async function auth(req, res) {
+  /**
+   * Implement your own security here.
+   *
+   * It's your responsibility to ensure that the caller of this endpoint
+   * is a valid user by validating the cookies or authentication headers
+   * and that it has access to the requested room.
+   */
+  const room = req.body.room;
+  const response = await authorize({
+    room,
+    secret,
+    // Corresponds to the UserMeta[id] type defined in liveblocks.config.ts
+    userId: "123",
+    groupIds: ["456"], // Optional
+    userInfo: {
+      // Optional, corresponds to the UserMeta[info] type defined in liveblocks.config.ts
+      name: "Ada Lovelace",
+      color: "red",
+    },
+  });
+  return res.status(response.status).end(response.body);
+}
+```
+
+<Banner title="Using a secret key with authorization">
+  To use the `authorize` method, you need to have your own back-end endpoint
+  which handles authentication. Your Liveblocks dashboard contains the secret
+  key required to use the `authorize` option.
+</Banner>
+
+To learn more about authorization and authentication in your Liveblocks
+application, check out our [Authentication Guide](/docs/guides/authentication).
+
+## How is MAU calculated using a public key? [#public-key]
+
+To take advantage of Liveblocks 1.0 pricing without authorization, you must
+update your Liveblocks packages to 1.0. We will count each connection as one MAU
+if you do not update your packages to 1.0. Once the packages are updated, a
+cookie will be set on the client to track when a new user connects to a room.
+This cookie will track MAU associated with a Liveblocks account and expire after
+30 days.
+
+Inside the `createClient` function, you will use your public key instead of a
+private key.
+
+```ts file="liveblocks.config.ts" highlight="4"
+import { createClient } from "@liveblocks/client";
+
+const client = createClient({
+  publicApiKey: "pk_prod_xxxxxxxxxxxxxxxxxxxxxxxx",
+});
+```
+
+To illustrate how counting MAU works without a `userId`, consider the following:
+
+If you have one user who connects to a Liveblocks room that does not enforce
+authorization, you will be billed for 1 MAU. That same person can reaccess a
+room the next day, and MAU would not increase. The cookie will expire at the end
+of the 30-day billing cycle, and the user will be counted as a new MAU.
+Liveblocks will count that anonymous user as 2 MAU if packages are not updated.
+
+## Development Use [#development-use]
```

Comment:
Should be "Development use"

Reply:
We could perhaps change this to reflect localhost usage even and explicitly say "Localhost usage isn’t included in your billing"? That way people can quickly scan this guide and get their most pressing question answered.